src/browsers/tob/shared/components/DotplotHeatmap.js

```diff
@@ -0,0 +1,218 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import { scaleBand } from 'd3'
+
+const tileSpacing = 0.05
+
+const DotplotHeatmap = ({
+  id,
+  data,
+  onClickFunction,
```

Comment:
Minor point, but can you rename `onClickFunction` to `onClick` to follow the typical convention for event handlers?